.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: reviewdog
+
+on: [push, pull_request]
+
+jobs:
+  mypy:
+    name: runner / mypy
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install -e .
+      - uses: tsuyoshicho/action-mypy@v3
+        with:
+          reporter: github-check
+          target: robotpy_installer
+          mypy_flags: --ignore-missing-imports
```

Comment:
Can't you add a `#type: ignore` comment to the pip import instead?

Reply:
That's probably actually a better idea.